integrations/opsgenie/.port/spec.yaml

```diff
@@ -13,7 +13,9 @@ configurations:
     required: true
     type: string
     sensitive: true
+    description: The API token used to authenticate requests to Opsgenie REST API. For more information on how to generate an API token, see [here](https://support.atlassian.com/opsgenie/docs/api-key-management/)
   - name: apiUrl
     type: string
     required: true
     default: https://api.opsgenie.com
+    description: The OpsGenie API URL. If not provided, the default value is https://api.opsgenie.com
```

Comment:
when it will be different? maybe we can suggest and add more information

Reply:
The api url differs based on the regions. Some clients are on the EU, some are on the US data centers